lib/menu.rb

```diff
@@ -0,0 +1,69 @@
+require 'json'
+
+class Menu
+
+     attr_accessor :customer_bill, :customer_order, :taxes, :sum, :amount
+     attr_reader :menu
+
+  def initialize
+       file = open("hipstercoffee.json")
+       json = file.read
+       @menu = JSON.parse(json)
+       @customer_order = []
+       @customer_bill = []
+       @total = 0
+       @taxes = 0
+       @quantity = 1
+
+  end
+
+  def order(booking, quantity)
+       @menu[0]["prices"][0].select do |key,value|
+          if booking == key
+       @customer_order.push key => value * quantity
+    end
+  end
+end
+
+
+  def prices
+      @customer_order.each do |hash|
+      @customer_bill << hash.values[0]
+  end
```

Comment:
Lint/BlockAlignment: `end` at 32, 2 is not aligned with `@customer_order.each do |hash|` at 30, 6.